Authentication/AuthenticationBootstrapper.swift

```diff
@@ -75,49 +76,87 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
             _window.rootViewController = UINavigationController(rootViewController: createLoginController())
         }
     }
+}
 
-    /**
+// MARK: - Login Functions
+public extension AuthenticationBootstrapper {
+    
+    /*
+         Creates the login controller to use for starting
+         the authentication process.
+         
+         - Returns: A valid login controller to use.
+     */
+    internal func createLoginController() -> LoginController {
+        let configuration = createLoginControllerConfiguration()
+        return LoginController(configuration: configuration)
+    }
+    
+    /*
```

Comment:
Why did you change this? If you remove the second *, the user of this class/function will not be able to see the description.